Wikipedia/Code/DescriptionEditViewController.swift

```diff
@@ -216,14 +215,14 @@ class DescriptionEditViewController: WMFScrollViewController, Themeable, UITextV
             DispatchQueue.main.async {
                 self.enableProgressiveButton(true)
                 guard let error = error else {
-                    self.editFunnel?.logWikidataDescriptionEditSaved(self.isEditingExistingDescription)
+                    self.editFunnel?.logWikidataDescriptionEditSaved(self.isEditingExistingDescription, language: language, revID: self.article?.revisionId)
                     self.delegate?.descriptionEditViewControllerEditSucceeded(self)
                     self.dismiss(animated: true) {
                         presentingVC?.wmf_showDescriptionPublishedPanelViewController(theme: self.theme)
                     }
                     return
                 }
-                self.editFunnel?.logWikidataDescriptionEditError(self.isEditingExistingDescription)
+                self.editFunnel?.logWikidataDescriptionEditError(self.isEditingExistingDescription, language: language, errorText: error.localizedDescription)
```

Comment:
Is this definitely what the schema calls for? I read it as only errors returned by the API, this could be a network error, JSON parse error, or any other app-side error. Using the localized string instead of a standard set of error code strings could also be an issue